src/main/java/org/fcrepo/transform/http/TransformNotFoundExceptionMapper.java

```diff
@@ -0,0 +1,26 @@
+package org.fcrepo.transform.http;
+
+import static javax.ws.rs.core.Response.status;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.fcrepo.transform.TransformNotFoundException;
+
+/**
+ * Handle TransformNotFoundExceptions
+ *
+ * @author whikloj
+ * @since 2016-03-25
+ */
+@Provider
+public class TransformNotFoundExceptionMapper implements ExceptionMapper<TransformNotFoundException> {
+
+    @Override
+    public Response toResponse(final TransformNotFoundException e) {
+        final String msg = e.getMessage();
+        return status(NOT_FOUND).entity(msg).build();
```

Comment:
Is a missing transform actually a 404? In that situation, the resource _was_ found, but the form in which it was supposed to be represented couldn't be constructed. I think this is a different response code.